CONTRIBUTING.md

```diff
@@ -6,9 +6,15 @@
 * [.NET Core SDK](https://www.microsoft.com/net/learn/get-started/windows)
     * Version driven by [global.json](src/Common/Commands.Common/global.json)
     * Cmdlets are designed to use .NET Core 2.0 SDK with .NET Standard 2.0
+* [Visual Studio Code](https://code.visualstudio.com/download)
+    * Experimental and optional, still need Visual Studio installed
+    * Install the following extensions:
+        * [C#](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp): `code --install-extension ms-vscode.csharp`
+        * [PowerShell](https://marketplace.visualstudio.com/items?itemName=ms-vscode.PowerShell): `code --install-extension ms-vscode.PowerShell`
 
-> If you plan to build with Configuration=Release which Delay Signs the build output, call `.\scripts\DisableStrongName.ps1`. 
-> Add the -Enable to renable strong name verification once developing.
+
+> If you plan to build with Configuration=Release which Delay Signs the build output, call `.\scripts\DisableStrongName.ps1`.
+> Add the -Enable to re-enable strong name verification once developing.
```

Comment:
"the -Enable parameter" or simply "-Enable"

Reply:
Ok